tools/IoTEdgeDevOps/DevOpsLib/VstsModels/VstsBuild.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace DevOpsLib.VstsModels
+{
+    using System;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
+
+    [JsonConverter(typeof(JsonPathConverter))]
+    public class VstsBuild
+    {
+        [JsonProperty("buildNumber")]
```

Comment:
Where do these "buildNumber", "_links.web.href", "status", "result", "queueTime", … come from? 
Could you please add a comment to the code referring to its source?

Reply:
will do